reset.css

```
@@ -0,0 +1,68 @@
+/* reset.css */
+
+/* 모든 요소의 margin과 padding을 0으로 초기화 */
+*,
+*::before,
+*::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box; 
+}
+
+/* 기본적인 폰트 설정 (웹사이트의 기본 폰트) */
+body {
+    font-family: 'Pretendard', sans-serif;
+    word-break: keep-all;
+}
+
+/* 텍스트 링크 스타일 초기화 */
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+/* 기본적으로 목록 스타일 초기화 */
+ul, ol {
+    list-style: none;
+}
+
+/* 테이블의 기본 스타일 초기화 */
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+
+/* 이미지 기본 스타일 초기화 */
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+/* 버튼 기본 스타일 초기화 */
+button {
+    font-family: 'Pretendard', sans-serif;
```

💊 제안
브라우저 스타일을 덮어씌우기 위해 이렇게 해주신 것 같은데 body 태그에 font-family를 명시해주셨으니 반복을 줄이기 위해
아래처럼 하셔서 상속받으시는 것을 추천드려요~

```suggestion
    font-family: inherit;
```